packages/react/src/feature-app-container.tsx

```diff
@@ -113,22 +131,37 @@ class InternalFeatureAppContainer extends React.PureComponent<
   }
 
   public render(): React.ReactNode {
-    if (!this.featureApp) {
+    if (!this.featureApp || this.state.hasError) {
       return null;
```

Comment:
Shouldn't we also throw the lifecycle error on the server instead of just rendering null here?

Reply:
[Error boundaries don't work on the server](https://reactjs.org/docs/error-boundaries.html) and we don't call `attachTo` there, so this won't happen there anyway.